src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -567,6 +567,10 @@
       " NL (You have !M! Curses.)"
     ]
   },
+  "stsjorbsmod:Possession": {
+    "NAME": "Possession",
+    "DESCRIPTION": "Possess target, dealing it's max hp in damage !M! times to another random target at the end of your turn. NL stsjorbsmod:Exert"
```

Comment:
I think maybe the wording should be more clear that it's a debuff on the enemy and that the enemy will be attacking its ally *instead* of performing its normal action next turn. Maybe something like:

```Apply !M! stsjorbsmod:Possessed. NL stsjorbsmod:Exert.```

...with a new keyword for possessed along the lines of:

```#yPossessed creatures will spend their next turn attacking random allies. They will attack a number of times equal to their #yPossessed stacks. Each attack deals damage equal to their max HP.```